bstrp_functions.sh

```diff
@@ -368,6 +390,7 @@ show_supported_features() {
   echo "W. Openwsman Support ...........$(print_feature_status OPENWSMAN_ENABLED)"
   echo "X. Azure Support ...............$(print_feature_status AZURE_ENABLED)"
   echo "Y. Systemd Support .............$(print_feature_status SYSTEMD_ENABLED)"
+  echo "Z. NanoFi Support ..............$(print_feature_status NANOFI_ENABLED)"
```

Comment:
We ran out of letters. I wonder what's going to happen to the next feature.

Reply:
We could discuss this when the next feature comes along, maybe we could continue with double letter menu entries like AA.

Reply:
or we could continue with Greek letters, like it is done with hurricanes :)  α βιτ ΣΗαλλενγινγ το τΥπε, βυτ ιτ ιζ ποσσιβλε

Reply:
@lordgamez It should be no problem to introduce options with more than one character. We read and match a whole line.
@fgerlits A bit challenging to read as well. :smile: